### DISCUSSION TITLE
Is there a way to use FastF1 API with JavaScript?

### DISCUSSION THREAD

**Discussion by dalton-f** [theOehrly/Fast-F1#643] [Q&A]
The documentation is mostly python, but I would like to use this API as it provides the most data within a website, similar to how F1 tempo does. I was wondering if there's an easy way to do this or if someone could point me in the right direction?

**Comment by theOehrly** [comment]
There is no way to directly use FastF1 wit JavaScript or similar. As far as I know, F1 Tempo uses FastF1 in their backend to pre-populated a database that they then access from their web server/frontend. Not sure how it works in detail.

**Comment by borolepratik** [comment]
@dalton-f you can try https://pyscript.net/.
